packages/rx_bloc_cli/mason_templates/bricks/lib_auth_apple/__brick__/lib/lib_auth_apple/models/apple_credentials_model.dart

```diff
@@ -0,0 +1,23 @@
+{{> licence.dart }}
+
+class AppleCredentialsModel {
+  AppleCredentialsModel(
+      {required this.authorizationCode,
+      this.userIdentifier,
+      this.firstName,
+      this.lastName,
+      this.email,
+      this.identityToken});
+
+  final String authorizationCode;
+  final String? userIdentifier;
```

Comment:
Same as above

Reply:
User data fields will only be set if this is the initial authentication between the app and Apple ID. Only authorisationCode will be not null for sure in all next calls.